test_stack.py

```diff
@@ -299,6 +345,17 @@ def test_bad_labels():
     code = '@label; push 2; jump @label; jump @label'
     list(parse_program(code))
 
+
+def test_bad_prefixes():
+    # A prefix without an instruction is an error.
+    with pytest.raises(ValueError):
+       list(parse_program('quiet'))
+    with pytest.raises(ValueError):
+       list(parse_program('cond'))
+    # Duplicated prefixes are not allowed.
+    with pytest.raises(ValueError):
+        list(parse_program('quiet quiet push 1'))
+
```

Comment:
You need tests that prevent parsing things where the "prefix" doesn't come first, like `add quiet`.